src/Nowruz/pages/Credentials/index.tsx

```diff
@@ -3,28 +3,55 @@ import { HorizontalTabs } from 'src/Nowruz/modules/general/components/horizontal
 
 import css from './credentials.module.scss';
 import { useCredentials } from './useCredentials';
+import { KYBModal } from 'src/Nowruz/modules/credentials/KYB';
+import { TopBanner } from 'src/Nowruz/modules/general/components/topBanner';
+import { Icon } from 'src/Nowruz/general/Icon';
+import { useState } from 'react';
 
 export const Credentials = () => {
-  const { tabs } = useCredentials();
-
+  const { tabs, verified, setOpenVerifiyAlert, openVerifiyAlert } = useCredentials();
+  const [displayVerifiedBanner, setDisplayVerifiedBanner] = useState(true);
```

Comment:
every time the user reloads the page, shows the success banner and every time reloading the user should press `dismiss`.
(like Snapp! bug :))
please ask B, to specify the requirement that is it ok to show every time?
 or we should set it in `local/session storage`